addons/base_automation/models/base_automation.py

```diff
@@ -486,6 +486,11 @@ def _get_filter_domain_fields(self):
 
     def _get_trigger_specific_field(self):
         self.ensure_one()
+        if (isinstance(self.id, models.NewId)
```

Comment:
I'm sure there is an actual way to do this in the ORM, though I fail to see why just passing the default values doesn't work.

In any case, this method is a helper and should not care whether the record is a new one or not,
for the same reason, it should not access the context that way, actual parameters are much more explicit.
And finally, you make it return a recordset of multiple records when the code only tolerates retunrning 1 record (-- and the method's name has the singular field)

I'd make the change you want in the compute methods of "trigger" and "trigger field_ids", even by adding a `@depends_context`

Reply:
Hello @kebeclibre,

Actually the issue comes from this recent `_onchange_trigger`. Updating this method would makes sense to you?